File: test_spot.js
const https = require('https');
https.get('https://open.spotify.com/episode/2lErM1P0U3NMHlQHOrnoKM', (resp) => {
  let data = '';
  resp.on('data', (c) => data += c);
  resp.on('end', () => {
    let match = data.match(/Creator Economy Live/i);
    console.log(match !== null);
  });
});

File: test_spot.py
import asyncio
from transcriber.whisper_transcriber import transcribe_url_async
async def main():
    u = await transcribe_url_async("https://open.spotify.com/episode/2lErM1P0U3NMHlQHOrnoKM")
    print(u)

asyncio.run(main())
